tests/pkg/lvgl/main.c

```diff
@@ -92,7 +92,7 @@ void sysmon_create(void)
 
     /* Create a chart with two data lines */
     chart = lv_chart_create(cont);
-    lv_obj_set_size(chart, hres / 2.5, vres / 2);
+    lv_obj_set_size(chart, hres * 10 / 25, vres / 2);
```

Comment:
```suggestion
    lv_obj_set_size(chart, hres * 10L / 25, vres / 2);
```
and 

105 in tests/pkg/lvgl/main.c
GitHub Actions / static-tests

keyword 'for' not followed by a single space